server/src/test/java/org/elasticsearch/action/admin/cluster/node/stats/NodeStatsTests.java

```diff
@@ -637,6 +638,8 @@ private static CommonStats createShardLevelCommonStats() {
         indicesCommonStats.getBulk().add(new BulkStats(++iota, ++iota, ++iota, ++iota, ++iota));
         indicesCommonStats.getShards().add(new ShardCountStats(++iota));
 
+        indicesCommonStats.getDenseVectorStats().add(new DenseVectorStats(++iota));
```

Comment:
Seems like this is nullable? Do we have any tests for the serialization of this that allow for it to be null?

Reply:
I added a dedicated test for CommonStats since it was missing anyway:
https://github.com/elastic/elasticsearch/pull/98275/commits/60b5e69ac5c0759d0fd79391d093527e4e690502